e2e/tests/init_test.go

```diff
@@ -912,6 +934,84 @@ func (s *suite) getServiceComputeUnitsPerRelay(serviceId string) uint64 {
 	return resp.Service.ComputeUnitsPerRelay
 }
 
+// getSharedParams returns the shared module parameters
```

Comment:
Can you generalize and reuse this helper?

![Screenshot 2025-03-03 at 2 27 02 PM](https://github.com/user-attachments/assets/77213582-d572-4950-9c89-4716d030538e)